src/storage/lock_manager.rs

```diff
@@ -10,9 +12,13 @@ pub struct Lock {
     pub hash: u64,
 }
 
+/// DiagnosticContext is for diagnosing problems about locks
 #[derive(Clone, Default)]
 pub struct DiagnosticContext {
+    /// the key we care about
```

Comment:
Prefer capitalize the first character in comments. The same to other comments.
```suggestion
    /// The key we care about
```